quadrat/sass/blocks/_table.scss

```diff
@@ -0,0 +1,20 @@
+.wp-block-table {
+	th {
+		font-weight: 500;
+		text-align: left;
```

Comment:
This interferes with Gutenberg's text align utility classes: 

<img width="1272" alt="Screen Shot 2021-04-29 at 12 18 10 PM" src="https://user-images.githubusercontent.com/5375500/116584112-dd538d80-a8cb-11eb-979b-7418a4e71ed2.png">

Could we remove it? Users still have the option of aligning the header to the left.

Reply:
I guess that's a question for @beafialho and @kjellr